### ISSUE TITLE
Integrate Fastly in the Adobe I/O deployment

### ISSUE THREAD

**Issue by kptdobe** [adobe/Adobe-Developer-Site#2]
** Goals **
Ground work for Helix to handle some of the Adobe I/O pages. This would require Fastly to be in the front of adobe.io. The exact list of todo has to be defined but should look like
- Create Fastly accounts and services
- Initiate DNS routing to Fastly

** Expected outcomes **
- Precise list of action items (to serve as documentation for next project)
- Fastly as CDN for Adobe I/O

**Comment by trieloff** [comment]
@kptdobe I assume that this is not for the existing site, but the new site. This means for now, we would only need the `new.adobe.io` domain name.

**Comment by trieloff** [comment]
The hardest part is probably the DNS on boarding, I will need help from @kgera here.

**Comment by kptdobe** [comment]
I think @davidnuescheler and @simonwex idea is to use Fastly in the front for Adobe I/O in general. But I let them confirm.

**Comment by trieloff** [comment]
https://adobeio.primordialsoup.life is now online on Fastly. Service ID: `60uIaEGMhHzvppobXGEPbS`

**Comment by trieloff** [comment]
The domain `www.adobe.io` is owned by another Fastly customer, I will try to figure out who it is.

**Comment by kgera** [comment]
@trieloff have you managed to track that? Also, on a side note - does fastly verify the domain ownership?

**Comment by lcrabb** [comment]
@trieloff any update on this?

**Comment by trieloff** [comment]
I've opened a support ticket with Fastly http://fastly.zendesk.com/hc/requests/100106 before I went on vacation and received a message that the domain is now available, but trying this morning, this does not seem to be the case.

I've reopened the ticket and will keep you updated.

@kgera I also noticed that while `curl https://adobeio.primordialsoup.life` gives the correct response, there seems to be some client-side redirect if the URL does not match `www.adobe.io`, which makes testing a bit difficult. Could we disable or relax this, so that we can test the next steps I'll describe below.

My plan of action is this:

1. get the domain ownership issue resolved with Fastly
2. add two domains to the service config: `www.adobe.io` and `www2.adobe.io` 
3. add `www2.adobe.io` and `www.adobe.io` to the Fastly shared TLS certificate. @lcrabb this will require us to add a DNS TXT record for both subdomains to verify ownership. Alternatively, we could put a static file on `www.adobe.io/…`
4. once `www2.adobe.io` is TLS-onboarded, we point the DNS record to Fastly, so that Fastly will serve `www2.adobe.io` traffic, while our existing setup will remain for `www.adobe.io`
5. we test everything
6. we repeat step 4 for `www.adobe.io`

**Comment by kgera** [comment]
Thanks @trieloff  @lcrabb Do you have domain ownership for www.adobe.io, my understanding was it is with @adrocknaphobia 's team.

**Comment by trieloff** [comment]
`www2.adobe.io` and `www.adobe.io` are now part of the Fastly service config.

**Comment by filmaj** [comment]
@simonwex @kgera should we keep this issue alive still? In JIRA ([DEVEP-1140](https://jira.corp.adobe.com/browse/DEVEP-1140)) I see a related issue. Also @simonwex is the milestone still relevant?

**Comment by kgera** [comment]
yes @filmaj this has to remain open. We are tracking redirects which is a sub-issue in DevRel JIRa board.

**Comment by filmaj** [comment]
@kgera could you post the link to the JIRA issue please?
